src/main/java/core/basesyntax/model/figure/Triangle.java

```diff
@@ -1,8 +1,27 @@
 package core.basesyntax.model.figure;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "circle")
 public class Triangle extends Figure {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```

Comment:
the same? maybe better move id to MappedSuperclass?

Reply:
fixed